doc/user/content/sql/create-source/postgres.md

```diff
@@ -170,9 +148,11 @@ CREATE CONNECTION pg_connection
 
 ```sql
 CREATE SOURCE mz_source
-FROM POSTGRES
-  CONNECTION pg_connection
-  (PUBLICATION 'mz_source');
+  FROM POSTGRES
+    CONNECTION pg_connection
+    (PUBLICATION 'mz_source')
```

Comment:
```suggestion
    CONNECTION pg_connection (PUBLICATION 'mz_source')
```

I think we should try to consistently format these like `CREATE TABLE`, either all one line, or with the open parens starting an indented block:

```
CONNECTION pg_connection (
    PUBLICATION 'mz_source',
    IDKSOMEOTHEROPTION 'we add someday'
)
```

Reply:
> IDKSOMEOTHEROPTION 'we add someday'

:blob-no:

Will make them consistently use one line for now.